docs/source/contribution/technical_steering_committee.md

```diff
@@ -74,7 +74,8 @@ Currently, the core Kedro team consists of:
 [Rashida Kanchwala](https://github.com/rashidakanchwala),
 [Sajid Alam](https://github.com/SajidAlamQB),
 [Stephanie Kaiser](https://github.com/stephkaiser),
-[Tynan DeBold](https://github.com/tynandebold) and
+[Tynan DeBold](https://github.com/tynandebold),
+[Marcin Zabłocki](https://github.com/marrrcin) and
```

Comment:
@marrrcin This list is alphabetical by first name, so you'll have to move yourself up a bit 🙂